api/src/accept_type.rs

```diff
@@ -15,25 +15,22 @@ pub enum AcceptType {
     Bcs,
 }
 
+/// Check that the accept type is one of the allowed variants. If there is no
+/// overriding explicit accept type, default to JSON.
+fn parse_accept(accept: &Accept) -> Result<AcceptType> {
+    if matches!(accept.0, BCS) {
+        Ok(AcceptType::Bcs)
+    } else {
+        Ok(AcceptType::Json)
+    }
+}
+
```

Comment:
Does this handle being sent more than one MIME type?  I'm not a particular expert in HTTP, but it looked like the previous code handled if more than one MIME type was sent.

Reply:
It appears that the code only checks if the `accept` parameter matches a specific MIME type, BCS. If it doesn't match, it defaults to JSON. It doesn't seem to handle the case where there are multiple MIME types specified in the `accept` header.